stdlib/public/Darwin/Foundation/String.swift

```diff
@@ -96,3 +96,23 @@ extension Substring : _ObjectiveCBridgeable {
 }
 
 extension String: CVarArg {}
+
+/*
+ This is on NSObject so that the stdlib can call it in StringBridge.swift
+ without having to synthesize a receiver (e.g. lookup a class or allocate)
+ */
+private extension NSObject {
+  @_effects(releasenone)
```

Comment:
Unclear what exactly I'm claiming is not being released here, but it does cause something to not be retain-released